src/content/guidelines/accessibility/overview.md

```diff
@@ -1,48 +1,142 @@
+However, _how you use Carbon_ also affects the accessibility of a product. Read this page to learn about accessibility in general. Then, while you're working, use the role-specific Carbon accessibility guidelines to help you make products that anyone can use.
 
 ## What does it mean for a product to be accessible?
 
 Accessible products:
 
-- Give everyone the same benefits regardless of disabilities a user may have
-- Can adapt to any user in any context.
+- Give every user the same benefits, regardless of disabilities
+- Can adapt to any user in any context
 
-Disabilities are a mismatch between one's ability and one's environment. They can be:
+Disabilities can be thought of as a mismatch between one's ability and one's environment. They can be:
 
-|               |                 |
+| Type          |     Description |
 |-------------  | ----------------|  
 | Situational   |     A person with typical vision might struggle to view their screen in a bright environment. Or, a person sitting in a library may be unable to watch a video with the audio turned on. |
 | Temporary     |     A person with a broken wrist may not be able to type but will regain the ability when healed.      |
-| Long Lasting  |     This is what people typically think of when they hear "disability" and includes the categories listed below.   |
+| Long-Lasting  |     This is what people typically think of when they hear "disability."    |
 
-Accessible design is not exclusively for a fixed subset of users. When we design for people with disabilities, _we are designing for everyone in every situation._
+Accessible design not only helps users with disabilities; it provides better user experiences for everyone.
 
-## Global Accessibility Standards
-
-- [World Wide Web Consortium](https://en.wikipedia.org/wiki/World_Wide_Web_Consortium) (W3C)'s Web Accessibility Initiative (WAI) is an effort to improve the [accessibility](https://en.wikipedia.org/wiki/Web_accessibility) of the [World Wide Web](https://en.wikipedia.org/wiki/World_Wide_Web) for people with [disabilities](https://en.wikipedia.org/wiki/Disability).
-- Web Accessibility Initiative (WAI) contributors create and maintain [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/) (WCAG) which is the global accessibility standard.
 
 ## Main Categories of Disabilities
 
-Quick jump
-- [Blind](/guidelines/accessibility/disabilities/#blind)
-- [Low Vision](/guidelines/accessibility/disabilities/#low-vision)
-- [Deaf and Hard of Hearing](/guidelines/accessibility/disabilities/#deaf)
-- [Physical Disabilities](/guidelines/accessibility/disabilities/#physical)
-- [Color Blindness](/guidelines/accessibility/disabilities/#color)
-- [Cognitive Disabilities](/guidelines/accessibility/disabilities/#cognitive)
```

Comment:
Overall looks good, was just wondering if we should leave this quick jump menu in so users can jump down to the specific sections?

Reply:
This was left over from when that content was on a separate page. This content isn't long enough to justify the shortcuts.

Reply:
gotcha :) makes sesne! merging in